app/services/analytics_events.rb

```diff
@@ -1,64 +1,128 @@
 # rubocop:disable Metrics/ModuleLength
 module AnalyticsEvents
-  # @identity.idp.event_name Account Reset
+  # @identity.idp.event_name Account Reset: cancel
+  # @identity.idp.previous_event_name Account Reset
+  # @param [String] user_id
+  # @param [String, nil] message_id from AWS Pinpoint API
+  # @param [String, nil] request_id from AWS Pinpoint API
+  # An account reset was cancelled
+  def account_reset_cancel(user_id:, message_id: nil, request_id: nil, **extra)
+    track_event(
+      'Account Reset: cancel',
```

Comment:
don't we still want these as constants?

Reply:
in all the previous PRs, we haven't been keeping the constants, what would we use them for?

Reply:
They get used here and in tests, so I could see the case to have constants. I'm comfortable either way though.

Reply:
Ok, for consistency I'm going to keep this one as is. The bummer is that constants don't get expanded inside YARD Tags so we'd need to find a way to work around that or still have some duplication.

If anybody got a proposal for how we can DRY these and still keep the event documentation working, I'm all ears